back-end/src/base/calls/calls.service.ts

```diff
@@ -35,7 +35,7 @@ export class CallsService {
 	}
 
 	async generateUrl(jwt: string) {
-		return `https://1b68-2a01-cb00-e91-b600-2d60-c9db-21f5-24f.eu.ngrok.io/calls/presence/` + jwt;
+		return `https://backend-codingtools.loule.me/calls/presence/` + jwt;
```

Comment:
Add a comment TODO to specify that it needs to be under a dotenv variable for production.

Reply:
Will be done